aio/content/guide/pipes.md

```diff
@@ -1,49 +1,51 @@
-# Transforming Data Using Pipes
+# Transformar Información Usando Pipes
 
-Use [pipes](guide/glossary#pipe "Definition of a pipe") to transform strings, currency amounts, dates, and other data for display.
-Pipes are simple functions you can use in [template expressions](/guide/glossary#template-expression "Definition of template expression") to accept an input value and return a transformed value. Pipes are useful because you can use them throughout your application, while only declaring each pipe once.
-For example, you would use a pipe to show a date as **April 15, 1988** rather than the raw string format.
+Usa los [pipes](guide/glossary#pipe "Definicion de pipe") para transformar strings, símbolos de monedas, fechas y otro tipo de información para mostrar.
+Los Pipes son simples funciones que puedes usar en [la plantilla de expresión](/guide/glossary#template-expression "Definicion de plantilla de expresión") los cuales aceptan un valor de entrada y retornan ese valor transformado.
+Los pipes son útiles porque puedes usarlos en toda la aplicación, habiéndolos declarado solo una vez.
+
+Por ejemplo, puedes usar un pipe para mostrar una fecha como **April 15, 1988** en lugar de un string sin formato.
 
 <div class="alert is-helpful">
 
-  For the sample app used in this topic, see the <live-example></live-example>.
+Para ver la aplicación de muestra utilizada en este tema, consulte <live-example></live-example>.
 
 </div>
 
-Angular provides built-in pipes for typical data transformations, including transformations for internationalization (i18n), which use locale information to format data.
-The following are commonly used built-in pipes for data formatting:
+Angular tiene incorporados pipes para la transformación de datos típicos, incluyendo transformaciones para la internacionalización (i18n), que utilizan información de la configuración regional para formatear los datos.
```

Comment:
me suena como raro la traducción de "for typical data transformations" a "para la transformación de datos típicos".

Yo lo que pondria seria:

"para las tipicas transformaciones de datos"